scipy/stats/tests/test_distributions.py

```diff
@@ -8368,6 +8368,28 @@ def test_fit_nu(self, loc, scale):
         assert_allclose(scale_est, scale_theo, rtol=1e-7)
 
 
+class TestAnglit:
+
+    @pytest.mark.parametrize("t, ref",
+                            [(-0.6, 0.9660195429836131),
+                             (0.6, 0.033980457016386835)])
```

Comment:
Adds a case that doesn't pass in main. (Otherwise, I would just remove the tests.)
```suggestion
                            [(-np.pi/4 + 1e-6, 0.999999999999),
                             (-0.6, 0.9660195429836131),
                             (0.6, 0.033980457016386835),
                             (np.pi/4 - 1e-6, 1.0000000001184104e-12)])
```